src/components/NavBar/GeneralSearch.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+/* eslint-disable */
+const GeneralSearch = ({ onSearch, onSubmit, searchValue }) => {
+  return (
+    <form className="form-inline" id="generalSearchForm">
+      <input 
+      class="form-control mr-sm-2"
+      id="generalSearch"
+      type="search" 
+      placeholder="Search" 
```

Comment:
Since we are limiting it to all dictionaries, I suggest that we make the placeholder more descriptive

Reply:
That will be taken care of in the next pull request thank you for the feedback